contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
> "Inconsistency" was the wrong word, I suppose I meant minor annoyance. Anyways, thanks to both of you for providing clarification.

You're welcome.  It definitely is an annoyance, and we could add more to the Makefile to help overcome that.  I've just been reluctant to do so since my goal was to keep the Makefile slim.  If you and others feel differently, then perhaps it's worthwhile to do.